cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -0,0 +1,137 @@
+import fauxfactory
+import pytest
+
+from cfme import test_requirements
+from cfme.rest.gen_data import users as _users
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.rest import assert_response
+from cfme.utils.wait import wait_for_decorator
+
+pytestmark = [test_requirements.report, pytest.mark.tier(3), pytest.mark.sauce]
+
+
+@pytest.fixture(scope="function")
+def rbac_api(appliance, request):
+    user, user_data = _users(
+        request, appliance, password="smartvm", group="EvmGroup-user"
+    )
+    yield appliance.new_rest_api_instance(
+        entry_point=appliance.rest_api._entry_point,
+        auth=(user[0].userid, user_data[0]["password"]),
+    )
+    appliance.rest_api.collections.users.action.delete(id=user[0].id)
+
+
+@test_requirements.rest
+@pytest.mark.tier(1)
+def test_non_admin_user_reports_access_rest(appliance, request, rbac_api):
+    """ This test checks if a non-admin user with proper privileges can access all reports via API.
```

Comment:
Is this test case requires ```appliance, request,```?
If not please remove.

Reply:
rbac_api requires appliance and request, but I think I can remove appliance.